addons/point_of_sale/views/res_config_settings_views.xml

```diff
@@ -145,6 +145,9 @@
                                     </div>
                                 </div>
                             </setting>
+                            <setting string="Closing entry with Product Lines" help="Use Journal entries to get different product by session">
```

Comment:
```suggestion
                            <setting string="Group by Product at Closing Entry" help="Activating this setting will group the sales lines by product instead of sales account in the automatically generated journal entry during closing of session.">
```
We need to duplicate the help and description here because we want to make sure the message is visible to the user.

Reply:
Done.